ansible/server-setup/roles/netdata/vars/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Variables for Netdata
+
+# Set Netdata Cloud claiming details. To find your `claim_token` and
+# `claim_room`, go to Netdata Cloud, then click on your Space's name in the top
+# navigation, then click on `Manage your Space`. Click on the `Nodes` tab in the
+# panel that appears, which displays a script with `token` and `room` strings.
+# Copy those strings into the variables below. `claim_url` should be
+# `https://app.netdata.cloud`. Read more:
+# https://learn.netdata.cloud/docs/agent/claim
+claim_token: XXXXX
```

Comment:
Did you forget to update these? Should they come from the env instead ?

Reply:
No, this is a variable file, and the tokens aren't fixed. Need to update the values before running the playbook, or passing the `-e` or `--extra-vars` flags when running the playbook to override the default value.